lib/Command/VerifySecretCommand.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * @author Semih Serhat Karakaya <karakayasemi@itu.edu.tr>
+ *
+ * Two-factor TOTP
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\TwoFactor_Totp\Command;
+
+use OCA\TwoFactor_Totp\Db\TotpSecretMapper;
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\IUserManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class VerifySecretCommand extends Command {
+
+	/** @var TotpSecretMapper */
+	private $secretMapper;
+
+	/** @var IUserManager */
+	private $userManager;
+
+	public function __construct(
+		TotpSecretMapper $secretMapper,
+		IUserManager $userManager
+	) {
+		parent::__construct();
+		$this->secretMapper = $secretMapper;
+		$this->userManager = $userManager;
+	}
+
+	protected function configure() {
+		$this
+			->setName('twofactor_totp:verifysecret')
+			->setDescription(
+				'Verify secrets of specified users or all users'
+			)
+			->addOption(
+				'all',
+				null,
+				InputOption::VALUE_NONE,
+				'Will affects all users secrets'
+			)
+			->addOption(
+				'uid',
+				'u',
+				InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+				'The user\'s uid is used. This option can be used as --uid "Alice" --uid "Bob"'
+			);
+	}
+
+	/**
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 */
+	protected function execute(InputInterface $input, OutputInterface $output) {
+		$allUsers = $input->getOption('all');
+		$uids = $input->getOption('uid');
+		if ($allUsers !== false) {
+			$this->secretMapper->setAllSecretsVerificationStatus(true);
+			$output->writeln("<info>All the secrets are verified</info>");
+		} else if (\count($uids) > 0) {
```

Comment:
see my comment above, because you made two commands there is more duplicated code